src/Pagerfanta/View/DefaultView.php

```diff
@@ -31,11 +33,17 @@ class DefaultView implements ViewInterface
     private $startPage;
     private $endPage;
 
+    /**
+     * @param TemplateInterface $template
```

Comment:
this does not add any value at all compared to the signature. It only duplicates knowledge (and with a mistake as the actual type is ``TemplateInterface|null``)